src/System.Security.Principal.Windows/pkg/System.Security.Principal.Windows.pkgproj

```diff
@@ -4,7 +4,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Security.Principal.Windows.csproj">
-      <SupportedFramework>net46;netcore50;netcoreapp1.0</SupportedFramework>
```

Comment:
not necesarilly, if you want the new surface area of the contract to be exposed in UWP then yes, but the package will still work in uap10.1 if you don't add it, it will just have the old surface area.